ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityActivityBlock.ts

```diff
@@ -0,0 +1,20 @@
+import { ActivityBlock } from './activityBlock';
+import { BlockType } from './blockType';
+
+enum ChildActivityRendering {
```

Comment:
Name of enum seems to indicate it might be an action of some sort.
Would suggest renaming to "ActivityRenderHintType" or something else like that.

Reply:
renamed.